application/Program.cs

```diff
@@ -12,7 +12,7 @@ public static void Main(string[] args)
 
 		    using (var amz = new AmazonSynchronizer(new Configuration()))
 		    {
-                amz.WithInterval(TimeSpan.FromMinutes(1))
+                amz.WithInterval(TimeSpan.FromMinutes(5))
```

Comment:
you put any interval you want 1 second or 10 hours it's up to you.
if the systems finds out that the interval isn't set , it will set the default value which is 5 hours